### DISCUSSION TITLE
Joining subs

### DISCUSSION THREAD

**Discussion by A-Winters** [SubtitleEdit/subtitleedit#4927] [General]
Is there a way to join two .ass files without losing the font styles and which lines they're assigned to, please?

**Comment by LeonCheung** [comment]
Either way below:

![1](https://user-images.githubusercontent.com/494788/112428686-81e72a80-8d76-11eb-936b-8d326f54574e.jpg)

or

![2](https://user-images.githubusercontent.com/494788/112428692-83b0ee00-8d76-11eb-900e-8c06fe351ded.jpg)

But currently you have to rename the styles with duplicated names before that.
